.github/workflows/windows_release.yml

```diff
@@ -14,29 +14,22 @@ env:
 jobs:
   windows_release:
     runs-on: windows-2019
+    # to move to windows-2022, just make sure to set the Visual Studio generator build to "17 2022"
     continue-on-error: ${{ matrix.allow_failure }}
     strategy:
       # fail-fast: Default is true, switch to false to allow one platform to fail and still run others
       fail-fast: false
       matrix:
-        name: [x64, x86, x64_hardened]
```

Comment:
I propose that we drop 32 bit Windows.

Reply:
OK, so we still have at least one interface developer building 32-bit packages on Windows.  I need to check whether they are actually linking to EnergyPlus or not.  If not, they may not necessarily need a 32 bit EnergyPlus.  I don't want to break anything though, so I am likely to back off of this...again...to get the other stuff in here merged.